resources/views/fields/upload.blade.php

```diff
@@ -18,14 +18,26 @@
         </div>
     @endif
 
-    {{-- Show the file picker on CREATE form. --}}
-    <input
-            type="file"
-            id="{{ $field['name'] }}_file_input"
-            name="{{ $field['name'] }}"
-            value="{{ old($field['name']) ? old($field['name']) : (isset($field['value']) ? $field['value'] : (isset($field['default']) ? $field['default'] : '' )) }}"
-            @include('crud::inc.field_attributes', ['default_class' =>  isset($field['value']) && $field['value']!=null?'form-control hidden':'form-control'])
-    >
+    <div class="js-parent-input">
+        @if (isset($field['value']) && $field['value'] !== null)
+            {{-- File already exists : put the path value to hidden file  --}}
+            <input
+                    type="hidden"
+                    id="{{ $field['name'] }}_file_input"
+                    name="{{ $field['name'] }}"
+                    value="{{ $field['value'] }}"
+                    @include('crud::inc.field_attributes', ['default_class' =>  'form-control']) />
+        @else
+            {{-- File not exists : display an input file --}}
+            <input
+                    type="file"
+                    id="{{ $field['name'] }}_file_input"
+                    name="{{ $field['name'] }}"
+                    value="{{ isset($field['default']) ? $field['default'] : '' }}"
```

Comment:
We should take advantage of this PR to remove this line. It seems that an `input[type=file]` can't have a default value.

https://stackoverflow.com/questions/2665058/set-default-value-for-a-input-file-form
> The file input type creates a field through which users can upload files from their local computer or network. The VALUE attribute specifies the name of the initial file, but it is typically ignored by browsers as a security precaution.

Instead we should implement the default value in the hidden field. Here is a try :
```blade
    <div class="js-parent-input">
        @if (!empty($field['value']) || !empty($field['default'])))
            {{-- File already exists : put the path value to hidden file  --}}
            <input
                    type="hidden"
                    id="{{ $field['name'] }}_file_input"
                    name="{{ $field['name'] }}"
                    value="{{ !empty($field['value']) ? $field['value'] : $field['default'] }}"
                    @include('crud::inc.field_attributes', ['default_class' =>  'form-control']) />
        @else
            {{-- File not exists : display an input file --}}
            <input
                    type="file"
                    id="{{ $field['name'] }}_file_input"
                    name="{{ $field['name'] }}"
                    @include('crud::inc.field_attributes', ['default_class' => 'form-control'])
            />
        @endif
    </div>
```

Reply:
We can't dot that. In your case if $field['default'] is not empty, the input file will nether displayed. 
But the default value can be added after input file (in hidden input) and the javascript will delete it if input file change (line 68)